resources/js/Pages/Collections/Components/CollectionsHeading/RefreshButton.tsx

```diff
@@ -2,32 +2,24 @@ import { useState } from "react";
 import { useTranslation } from "react-i18next";
 import { IconButton } from "@/Components/Buttons";
 import { Tooltip } from "@/Components/Tooltip";
-import { useAuthorizedAction } from "@/Hooks/useAuthorizedAction";
 import { useToasts } from "@/Hooks/useToasts";
 import { isTruthy } from "@/Utils/is-truthy";
 
 export const RefreshButton = ({ wallet }: { wallet: App.Data.Wallet.WalletData | null }): JSX.Element => {
-    const [loading, setLoading] = useState(false);
     const [disabled, setDisabled] = useState(false);
     const { t } = useTranslation();
 
-    const { signedAction } = useAuthorizedAction();
     const { showToast } = useToasts();
 
     const refresh = (): void => {
-        void signedAction(async () => {
-            setLoading(true);
+        setDisabled(true);
 
-            await window.axios.post(route("refresh-collections"));
+        void window.axios.post(route("refresh-collections"));
```

Comment:
you can replace `signedAction` with `authenticatedAction` (that comes from the same hook) so it still handles the case the session expired (so if session is lost it will require user to sign again)